velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -475,16 +475,19 @@ MaterializedRow getColumns(
   return columns;
 }
 
-void printRow(const MaterializedRow& row, std::ostringstream& out) {
-  out << row[0];
+void printRow(
+    const MaterializedRow& row,
+    std::ostringstream& out,
+    const TypePtr& type) {
+  out << row[0].toJson(type);
   for (int32_t i = 1; i < row.size(); ++i) {
-    out << " | " << row[i];
+    out << " | " << row[i].toJson(type);
```

Comment:
let's remove the `<<` overload from Variant.h

Reply:
Fixed.